.github/workflows/concrete_python_benchmark.yml

```diff
@@ -61,8 +61,7 @@ jobs:
             set -e
             rm -rf /build/*
 
-            export PYTHON=${{ format('python{0}', matrix.python-version) }}
-            echo "Using $PYTHON"
+            export PYTHON=python
```

Comment:
Better use a specific version here (one of "3.8", "3.9", "3.10", "3.11", "3.12")